x-pack/plugins/ml/public/application/model_management/expanded_row.tsx

```diff
@@ -20,37 +20,44 @@ import {
   EuiTabbedContent,
   EuiTabbedContentTab,
   EuiTitle,
+  useEuiPaddingSize,
 } from '@elastic/eui';
 import { FormattedMessage } from '@kbn/i18n-react';
 import { FIELD_FORMAT_IDS } from '@kbn/field-formats-plugin/common';
 import { isPopulatedObject } from '@kbn/ml-is-populated-object';
 import { isDefined } from '@kbn/ml-is-defined';
 import type { ModelItemFull } from './models_list';
 import { ModelPipelines } from './pipelines';
-import { AllocatedModels } from '../nodes_overview/allocated_models';
-import type { AllocatedModel } from '../../../../common/types/trained_models';
-import { useFieldFormatter } from '../../contexts/kibana/use_field_formatter';
+import { AllocatedModels } from '../memory_usage/nodes_overview/allocated_models';
+import type { AllocatedModel } from '../../../common/types/trained_models';
+import { useFieldFormatter } from '../contexts/kibana/use_field_formatter';
 
 interface ExpandedRowProps {
   item: ModelItemFull;
 }
 
-const badgeFormatter = (items: string[]) => {
-  if (items.length === 0) return;
-  return (
-    <div>
-      {items.map((item) => (
-        <EuiBadge key={item} color="hollow">
-          {item}
-        </EuiBadge>
-      ))}
-    </div>
-  );
+const useBadgeFormatter = () => {
+  const xs = useEuiPaddingSize('xs');
+
+  function badgeFormatter(items: string[]) {
+    if (items.length === 0) return;
+    return (
+      <div>
+        {items.map((item) => (
+          <span css={{ marginRight: xs! }} key={item}>
```

Comment:
nit: do you need a `span` wrapper?

Reply:
there is a rule which forces a list of components using the class `euiBadge` to have a starting margin of `4px`, causing this indentation when the row wraps.
![image](https://user-images.githubusercontent.com/22172091/216989210-f24284b4-92db-494a-8075-509e1b6f5ca8.png)

by wrapping the badge in a span and applying the padding to the end of the items, it looks better IMO
![image](https://user-images.githubusercontent.com/22172091/216989438-4ff558c8-3d2b-4765-8ec9-bc69efff37aa.png)

Applying the css override to the badges themselves does not stop the indentation.